CribblyBackend.UnitTests/ServiceTests/StandingsServiceTests.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Data.Common;
+using System.Threading.Tasks;
+using CribblyBackend.Controllers;
+using CribblyBackend.DataAccess.Models;
+using CribblyBackend.DataAccess.Repositories;
+using CribblyBackend.Services;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Moq.Dapper;
+using Xunit;
+
+namespace CribblyBackend.UnitTests
+{
+    public class StandingsServiceTests
+    {
+        private readonly StandingsService StandingsService;
+        private readonly Mock<IGameRepository> mockGameRepository;
+
+        private Team team1 = new Team() { Id = 1, Name = "team1"};
+        private Team team2 = new Team() { Id = 1, Name = "team2"};
+
+        public StandingsServiceTests()
+        {
+            mockGameRepository = new Mock<IGameRepository>();
+            StandingsService = new StandingsService(mockGameRepository.Object);
+        }
+        [Fact]
+        public async Task Calculate_ShouldReturnListsForTeamGameProperties()
+        {
+            Game game = new Game(){ Teams = new List<Team>(){team1, team2}};
+            var games = new List<Game>(){game};
+            mockGameRepository.Setup(x => x.GetByTeamId(team1.Id)).ReturnsAsync(games);
+            var result = await StandingsService.Calculate(team1);
+            Assert.IsType<List<Game>>(result.BracketGames);
+            Assert.IsType<List<Game>>(result.PlayInGames);
+        }
+        [Fact]
+        public async Task Calculate_ShouldReturnTeamWithOneWinIfTheyHaveLoss()
```

Comment:
🤔 I'm a bit confused by this test name. Which team is supposed to have one win? Why are we not asserting wins?

Reply:
As far as I can see you're asserting that the team has one loss, so I'd expect the test to be named something more like `Calculate_ShouldReturnTeamWIthOneLossIfTheyHaveLoss()`